aws-iot-scheduledaudit/src/main/java/com/amazonaws/iot/scheduledaudit/Translator.java

```diff
@@ -97,11 +96,6 @@ static BaseHandlerException translateIotExceptionToCfn(IotException e) {
             .collect(Collectors.toSet());
   }
 
-  static String translateAuditFrequencyToCfn(AuditFrequency auditFrequency) {
-
-    return auditFrequency == null? null : auditFrequency.toString();
-  }
-
   static String translateDayOfTheWeekToCfn(DayOfWeek dayOfWeek) {
```

Comment:
Sorry, I should've mentioned this in the previous revision - same comment here about using the `*AsString` method instead